using procedures and functions.cpp

De-Len opened the pull request.

De-Len committed:
using procedures and functions.cpp

Silmarill reviewed (changes requested):
- sum vs count
- single-letter

De-Len committed:
Update using procedures and functions — копия.cpp

De-Len committed:
Update using procedures and functions — копия.cpp

Done indentation by 2 spaces and fixed single-letter into lower_case_with_underscores

De-Len committed:
Update using procedures and functions — копия.cpp

(ver 3.1) Done indentation by 2 spaces and fixed single-letter into lower_case_with_underscores
